mpi.rst

```diff
@@ -257,6 +259,24 @@ If the host MPI is Open MPI, the definition file looks like:
 
       mpirun -np 2 -mca pml ucx -x UCX_TLS=sysv,ib apptainer exec $MY_CONTAINER ./a.out
 
+Using --sharens mode
+====================
+
+Default in unprivileged user namespace mode. Each process spawned by MPI runs in its own user namespace. This will 
```

Comment:
This looks like it is saying that `--sharens` is the default, but that's not what you mean.  Change the first two sentences to
```
By default in unprivileged user namespace mode each {command} command runs in its own user namespace.
```
I left MPI out from this because it is in general true, so then start the next sentence with
```
This causes problems for MPI runs, as observed...
```

Reply:
thanks! reflected in the latest commit.